### ISSUE TITLE
Remove AMP does not work

### ISSUE THREAD

**Issue by cbwee** [kiwibrowser/src.next#225]
**Describe the bug**
I have enabled Remove AMP, but it does not work when I opened `amp.reddit.com` links from Google search.

**To Reproduce**
1. Enable Remove AMP in settings
2. Open `https://www.google.com/search?q=amp+reddit` in Kiwi Browser.
3. Open the second search result.
4. The AMP page 
`https://amp.reddit.com/r/explainlikeimfive/comments/ecrzvp/eli5_what_are_amp_pages_and_whats_bad_about_them/`
 is opened.

**Expected behavior**
The browser should remove AMP and show the normal version of the webpage.
`https://www.reddit.com/r/explainlikeimfive/comments/ecrzvp/eli5_what_are_amp_pages_and_whats_bad_about_them/`

**Screen recording**

https://user-images.githubusercontent.com/34395942/127593891-c41f63c3-d228-4e6f-979d-f0087cb36902.mp4

**Smartphone (please complete the following information):**
 - Device: Xiaomi POCO M3
 - Android Version : Android 10
 - Kiwi Browser Version 93.0.4577.18

**Comment by kiwibrowser** [comment]
Thank you very much, your reports are very very nicely done ! I will prepare a fix for AMP in the coming days

**Comment by DUOLabs333** [comment]
I think I found the issue: in PersonalizeResults, when you get shouldRemoveAmp, the getBoolean for "avoid_amp_websites" has a second parameter of `true` instead of `false`.

**Comment by DUOLabs333** [comment]
@kiwibrowser Some websites, for example the register.com uses AMP, not amp in their urls, so you have to account for upper and lowercase.

**Comment by KaKi87** [comment]
There's also AdguardTeam/DisableAMP that work well as a usercript (with violentmonkey/violentmonkey for example).

**Comment by karl-pae** [comment]
Hi, download the latest build from here:
https://github.com/kiwibrowser/src.next/releases

and revert if the issue still occurs, else you can close this issue.
